### PR TITLE
Add warning if a node is tagged railway=flat_crossing

### LINKED ISSUE #158: Validator railway=flat_crossing -> railway=railway_crossing
Validator should show an "error" if a node is tagged `railway=flat_crossing` instead of `railway_crossing`.

### PR DESCRIPTION
**Pull request by Nakaner**
This fixes #158.

### CODE REVIEW

**Comment by DerDakon** [comment]
Drop the "fixRemove", it is not needed since you add it back with a different value anyway.

**Comment by Nakaner** [comment]
@DerDakon Thank you. I removed it.

File: validator/openrailwaymap.validator.mapcss [hunk 1]
```diff
@@ -372,6 +372,16 @@ area[railway=level_crossing]
 	assertNoMatch: "node railway=crossing";
 }
 
+/* railway=flat_crossing is deprecated, use railway=railway_crossing instead. */
+node[railway=flat_crossing]
+{
+	throwError: "railway=flat_crossing is deprecated, use railway=railway_crossing instead.";
+	assertMatch: "way railway=flat_crossing" ;
```
Comment (rurseekatze) [inline validator/openrailwaymap.validator.mapcss]: You should use `node` instead of `way`.

**Comment by Nakaner** [comment]
I fixed the wrong `assertMatch`. You can merge this pull request now.

**Pull request merged**